doc/examples/denoise_patch2self.py

```diff
@@ -68,10 +68,18 @@
 hardi_fname, hardi_bval_fname, hardi_bvec_fname = get_fnames('stanford_hardi')
 data, affine = load_nifti(hardi_fname)
 bvals = np.loadtxt(hardi_bval_fname)
-denoised_arr = patch2self(data, bvals, shift_intensity=True,
-                          clip_negative_vals=False)
+denoised_arr = patch2self(data, bvals, model='ols', shift_intensity=True,
+                          clip_negative_vals=False, b0_threshold=100)
```

Comment:
Sorry -- one more: why is `b0_threshold` set to a higher value than the `gradient_table` default?

Reply:
Okay... Set it back to 50, which is the default (even in the codebase). I added a note that people should change it if they are working with data such as HCP 👍🏽